app.js

```diff
@@ -1,51 +1,53 @@
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
+const bookSection = document.querySelector('#library');
+const titleText = document.querySelector('#title');
+const authorText = document.querySelector('#author');
+const form = document.querySelector('form');
 
-let bookKLibrary = [];
-// eslint-disable-next-line
-function bookHTML() {
-  const bookSection = document.querySelector('#library');
-  bookSection.innerHTML = '';
-  bookKLibrary.forEach((n) => {
-    bookSection.innerHTML += `<div>
-        <p>${n.title}</p>
-        <p>${n.author}</p>
-        <button type='button' onclick=remove('${n.title}','${n.author}') >Remove</button>
-        <hr>
-    </div>`;
```

Comment:
- [ ] From the images below you can see that you are missing the `by` word used in the wireframe please work on correcting your project design :+1:
- wireframe
![mb](https://github.com/BenMKT/Awseome-Books-Project/assets/89055502/436c41fd-c55e-4459-b9ce-77644eb72594)
- your design
![yd](https://github.com/BenMKT/Awseome-Books-Project/assets/89055502/15b66991-22a7-4a9c-b03a-e3ad49d4d849)

Reply:
Thanks for the review. Will work on it.

Reply:
-Added a `<p>` tag in the `<div class='bookDetail">` to resolve bug